js-webapp-async/README.md

```diff
@@ -1,22 +1,26 @@
 # Introduction
-In a [previous blogpost](../js-webapp/README.md) we discussed how to run c++ code on the web using Javascript.
+In an [earlier blogpost](../js-webapp/README.md) we discussed how to run c++ code on the web using Javascript. We created a webapp that executed some C++ code and then showed the result. While the page was running the C++ code, the page was blocked and unresponsive. That was not a problem then, because the computation done in the code was very quick. Blocked UI becomes a problem when we are performing tasks that take a bit longer to run. 
 
-We created a webapp that executed some C++ code and showed the result. While the page was running the C++ code, the page was blocked and unresponsive. That was not a problem then, because the computation done in the code was tiny. This becomes a problem when we are performing long running tasks instead. How to prevent blocking when running long running tasks in c++?
+How to prevent blocking when running long running tasks in c++?

```

Comment:
This sentence needs to be rephrased. Maybe combine with the next sentence?

Reply:
I agree that this could be phrased a bit smoother I guess. This is not the first iteration of this piece of text so it's kinda hard to improve it further. I guess for you, too, otherwise, you would have suggested something I suppose.

What specifically do you not like about this? Maybe even better to merge it like it is now, and improve this later.